src/main/resources/org/jenkinsci/plugins/workflow/libs/LibraryCachingConfiguration/config.jelly

```diff
@@ -31,5 +31,7 @@ THE SOFTWARE.
     <f:entry title="${%Versions to exclude}" field="excludedVersionsStr">
         <f:textbox />
     </f:entry>
-    <f:validateButton title="${%Clear cache}" progress="${%Clearing...}" method="clearCache" with="name" />
+    <j:if test="${h.hasPermission(app.ADMINISTER)}">
+        <f:validateButton title="${%Clear cache}" progress="${%Clearing...}" method="clearCache" with="name" />
+    </j:if>
```

Comment:
The backend has always required `ADMINISTER` permission, but we used to show this button on folder-level libraries to anyone with Job/Configure permission, which is just confusing.